File: Alumnos_reprobados_o_no.rb
continuar="s"
while continuar=="s"
    puts "Bienvenido a calcular si un alumno esta reprobado o no \n"
    puts "Ingrese calificacion del primer parcial"
    n1=gets.to_f
    puts "Ingrese calificacion del segundo parcial"
    n2=gets.to_f
    n3=n1+n2
    n3=n3/2
    if n3<7
        puts "El alumno saco: ", n3
        puts "El alumno esta reprobado"
    else
        puts "El alumno saco: ", n3
        puts "El alumno esta aprobado"
    end
    puts "¿Desea realizar otra operacion? s=Si n=No"
    continuar=gets.chomp
end

File: Alumnos_reprobados_o_no.py
continuar="1"
while continuar=="1":
    print("Bienvenido a la calculadora de ver si un alumno esta reprobado o aprobado")
    n1=int(input("Escriba la calificacion del primer parcial: "))
    n2=int(input("EScriba la calificacion del segundo parcial: "))
    n3=n1+n2
    n3=n3/2
    if n3<=7:
        print("El alumno saco: ", n3)
        print("El alumno esta reprobado")
    else:
        print("El alumno saco: ", n3)
        print("El alumno esta aprobado")
    continuar=input("¿Desea calcular otra? 1=Si 2=No")
